src/mlang/backend_compilers/bir_to_dgfip_c.ml

```diff
@@ -15,6 +15,20 @@ open Mir
 
 let none_value = "m_undefined"
 
+module Error = struct
+  type t = {
```

Comment:
This type should actually replace `Mir.Error`, because I didn't realize that the list of strings has actually a strict meaning.